CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to REXPX
+As an open source project, RESPX welcomes contributions of many forms.
+
+Examples of contributions include:
+
+* Code patches
+* Documentation improvements
+* Bug reports and patch reviews
+
+## Running Tests
+Tests reside in the `tests/` directory.
+You can run tests with the [Task](https://taskfile.dev/installation/) tool from the root of the project.
+
+You'll need a virtual environment setup.
+
+* `virtualenv venv`
+* `source venv/bin/activate`
+* `pip install httpx flask starlette pytest pytest-cov pytest-asyncio`
```

Comment:
These steps is taken care automatically by the internal `tools` task that `test` depends on.

Reply:
I get this when I try to run the `tools` task. And when I try to run the `tests` command without installing deps.

```
➜  respx git:(hm-add-contributing) ✗ task tools
task: Task "tools" is internal
➜  respx git:(hm-add-contributing) ✗ task test
nox > Running session test-3.11
nox > Re-using existing virtual environment at .nox/test-3-11.
nox > Program pytest not found.
nox > Session test-3.11 failed.
task: Failed to run task "test": exit status 1
```